tests/main/ubuntu-core-gadget-config-defaults/task.yaml

```diff
@@ -17,6 +17,13 @@ prepare: |
         echo "This test needs test keys to be trusted"
         exit
     fi
+
+    if systemctl -l | MATCH -v "$SERVICE.service"; then
```

Comment:
Is this the case only on uc18? If so, should we have an extra conditional to only skip it there? I fear we might be skipping this test where not expected without noticing.

Reply:
@stolowski the only one system with not rsyslog is uc18 but the idea of teh test is to skip it when syslog is not defined, it could also happen on boards and perhaps in uc20.

Reply:
Shouldn't we then disable uc18 via systems: ? I'd rather disable systems selectively when we learn something is not there, instead of a general logic, because then we might stop testing these bits without knowing.

Reply:
agree on this.